addons/im_livechat/migrations/10.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    env['ir.model.data'].search([
```

Comment:
You have to remove the records, not the XML-IDs:
```
xml_ids = [
    'im_livechat.ir_action_client_open_livechat_menu',
    'im_livechat.ir_cron_remove_empty_session',
]
for xml_id in xml_ids:
    self.env.ref(xml_ids).unlink()
```
I have seen that the the record referenced from base module is not needed.